src/content/app/tools/blast/blast-download/createBlastCSVTable.ts

```diff
@@ -0,0 +1,184 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { BlastJobResult } from 'src/content/app/tools/blast/types/blastJob';
+
+export const createCSVForGenomicBlast = (blastJobResult: BlastJobResult) => {
+  const columnNames = [
+    'E-value',
+    'Hit length',
+    '% ID',
+    'Score',
+    'Genomic location',
+    'Hit orientation',
+    'Hit start',
+    'Hit end',
+    'Query start',
+    'Query end'
+  ];
+
+  const rows = [columnNames] as Array<string | number>[];
+
+  for (const hit of blastJobResult.hits) {
+    for (const hsp of hit.hit_hsps) {
+      const {
+        evalue,
+        hitLength,
+        percentId,
+        score,
+        hitOrientation,
+        hitStart,
+        hitEnd,
+        queryStart,
+        queryEnd
+      } = getCommonFields({ blastHsp: hsp });
+      const genomicLocation = `${hit.hit_acc}:${hsp.hsp_hit_from}-${hsp.hsp_hit_to}`;
```

Comment:
Should the position of `from` and `to` values be based on whichever is higher?

Reply:
✅